### DISCUSSION TITLE
Plans for V14?

### DISCUSSION THREAD

**Discussion by JohnBergman** [leekelleher/umbraco-contentment#343] [Discussion]
I read that V13 of Umbraco is going to change the backend significantly, are there plans to continue working on this package to support V13 when it arrives?

**Comment by leekelleher** [comment]
Hi @JohnBergman, do you mean v14 with the new backoffice?

It is still early days in terms of package development for Bellissima (the codename for the new backoffice). But I am outlining my intentions and progress on ticket #316.
